testsuite/features/secondary/min_salt_formulas.feature

```diff
@@ -10,9 +10,11 @@ Feature: Use salt formulas
    Scenario: Log in as admin user
       Given I am authorized for the "Admin" section
 
-   Scenario: Install the locale formula package on the server
+   Scenario: Install the locale formula package on the server and enable repositories on the minion
```

Comment:
no need to put it in scenario title, it's an implementation detail IMHO

Reply:
Removed in both cases